doc/src/docs/EngineeringReference/13c-EncyclopaedicRefs.md

```diff
@@ -5362,13 +5373,1052 @@ where:
+
+Given <span>$T_{fs}$</span> and <span>$T_e$</span>, <span>$SH$</span> can be determined using the equation shown in Step 1.3.
+
+* If <span>$SH$</span> > 15°C, Set <span>$SH$</span> as 15°C. On/Off control strategy may be implemented when needed.
+
+#####Step 3c. 3: Total power consumption of the indoor unit
+
+The power consumption of the indoor unit comes from the fan operations. This can be calculated using the existing VAV fan model in EnergyPlus. Please refer to the current EnergyPlus Engineering Reference for more details.
+
+#### *Modeling of the indoor unit (I/U) - Part II - Heating Mode*
+
+#####Step 3h.1:  Update air flow rate for each indoor unit.
+
+1)	Calculate coil surface temperature for all the indoor units:
+
+<div>$$T_{fs}=T_c-[A_h\cdot SC^2+B_h\cdot SC+C_h]$$</div>
+
+Where
+
+<span>$T_c$</span>	condensing temperature decided in the outdoor unit calculations (°C)
+
+<span>$T_{fs}$</span>	coil surface temperature (°C)
+
+<span>$SC$</span>	subcooling degrees decided in the outdoor unit calculations (°C)
+
+2)	Calculate the required air flow rate <span>$G_a$</span> for each indoor unit:
+
+<div>$$G_a=Q_{in,total}/[(T_{fs}-T_{coil,in})\times{(1-BF)}\times{\rho_{in}}]$$</div> 
+
+Where
+
+<span>$Q_{in,total}$</span>	total cooling load for the zone (kW)
+
+<span>$T_{coil,in}$</span>	temperature of the suction air of the indoor unit (°C)
+
+<span>$\rho_{in}$</span>density of indoor air,<span>$f_{\rho}(T_{in},W_{in})$</span>   (kg/m<sup>3</sup>)
+
+3)	Decide whether to modify <span>$SH$</span> for further indoor unit capacity reduction
+
+* If <span>$G_a$</span> < <span>$G_{a,min}$</span> (e.g., <span>$0.7\times$</span><span>$G_{a,rate}$</span>) go to Step 3h. 2
+
+* Else, directly go to Step 3h. 3 
+
+##### Step 3h.2:  Modify <span>$SC$</span> to modify the indoor unit capacity
+
+Set <span>$G_a$</span> at its minimum value (<span>$0.7\times$</span><span>$G_{a,rate}$</span>).
+
+The required coil surface temperature <span>$T_{fs}$</span> can be determined as described in Step 2.
+
+Given <span>$T_{fs}$</span> and <span>$T_e$</span>, <span>$SC$</span> can be determined using the equation shown in Step 1.3.
+
+* If <span>$SC$</span> > 20°C, Set <span>$SC$</span> as 20°C. On/Off control strategy may be implemented when needed.
+
+##### Step 3h.3: Total power consumption of the indoor unit
+
+Calculate electric power consumption by the indoor unit using the existing VAV fan model in EnergyPlus. Please refer to the current EnergyPlus Engineering Reference for more details.
+
+#### Additional energy consumption by defrost and crankcase heater 
+
+There may be additional energy consumption due to the defrost operation and crankcase heater operation. These components have no impact on the heat pump operations. The calculation methods in the VRF_FluidTCtrl model are the same as those in VRF_SysCurve model. Please refer to the VRF_SysCurve section for more details.
+
```

Comment:
Yes, we could move all these similar routines to a common subroutine. I 
won't make you do that here.

On 8/27/2015 2:55 PM, Rongpeng Zhang wrote:
>
> In doc/src/docs/EngineeringReference/13c-EncyclopaedicRefs.md 
> <https://github.com/NREL/EnergyPlus/pull/5026#discussion_r38132816>:
>
> > +Set <span>$G_a$</span> at its minimum value (<span>$0.7\times$</span><span>$G_{a,rate}$</span>).
> > +
> > +The required coil surface temperature <span>$T_{fs}$</span> can be determined as described in Step 2.
> > +
> > +Given <span>$T_{fs}$</span> and <span>$T_e$</span>, <span>$SC$</span> can be determined using the equation shown in Step 1.3.
> > +
> > +* If <span>$SC$</span> > 20°C, Set <span>$SC$</span> as 20°C. On/Off control strategy may be implemented when needed.
> > +
> > +##### Step 3h.3: Total power consumption of the indoor unit
> > +
> > +Calculate electric power consumption by the indoor unit using the existing VAV fan model in EnergyPlus. Please refer to the current EnergyPlus Engineering Reference for more details.
> > +
> > +#### Additional energy consumption by defrost and crankcase heater
> > +
> > +There may be additional energy consumption due to the defrost operation and crankcase heater operation. These components have no impact on the heat pump operations. The calculation methods in the VRF_FluidTCtrl model are the same as those in VRF_SysCurve model. Please refer to the VRF_SysCurve section for more details.
> > +
>
> Rich, do you think it would be better to create a separated method for 
> such calculations? Also for crankcase heater calculations.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/NREL/EnergyPlus/pull/5026/files#r38132816>.
>

-- 
Richard Raustad
Florida Solar Energy Center
1679 Clearlake Road
Cocoa, FL 32922
Ph: (321)638-1454
http://www.fsec.ucf.edu/en/

Program Director
Electric Vehicle Transportation Center
http://evtc.fsec.ucf.edu/